prov/efa/src/efa_shm.c

```diff
@@ -143,7 +143,6 @@ void efa_shm_info_create(const struct fi_info *app_info, struct fi_info **shm_in
 	if (ret) {
 		EFA_WARN(FI_LOG_CORE, "Disabling EFA shared memory support; failed to get shm provider's info: %s\n",
 			fi_strerror(-ret));
-		efa_env.enable_shm_transfer = 0;
```

Comment:
Could you elaborate the reason `fi_getinfo` failed for sm2? Is there a way to make it work?

Reply:
I see. Thanks!